### ISSUE TITLE
ucp | create a local fetch directory if it doesn't exist - errors out.

### ISSUE THREAD

**Issue by vvb** [contiv/ansible#79]
When scheduler_provider = "ucp-swarm", the following error is seen 

```
TASK: [ucp | create a local fetch directory if it doesn't exist] **************
failed: [cluster-node1 -> 127.0.0.1] => {"failed": true, "parsed": false}
[sudo via ansible, key=pinebvdaownqyquvfpxaengckhfixffn] password:


FATAL: all hosts have already failed -- aborting
```

**Comment by vvb** [comment]
with ansible 2.0, the error msg is clearer..

```
TASK [ucp : create a local fetch directory if it doesn't exist] ****************
fatal: [cluster-node1 -> localhost]: FAILED! => {"changed": false, "failed": true,
 "module_stderr": "sudo: a password is required\n", "module_stdout": "", "msg": "MODULE FAILURE", "parsed": false}
```

**Comment by vvb** [comment]
resolved by #80
